src/commands/force/user/create.ts

```diff
@@ -57,6 +57,23 @@ export class UserCreateCommand extends SfdxCommand {
   private user: User;
   private successes: SuccessMsg[] = [];
   private failures: FailureMsg[] = [];
+  private authInfo: AuthInfo;
+
+  /**
+   * removes fields that cause errors in sfdx-core's userCreate method
```

Comment:
Not really "in sfdc-core's userCreate method". It is the Salesforce API that doesn't support the flags, hence the method name.